framework/Models/Model.py

```diff
@@ -296,7 +294,7 @@ def _handleInput(self, paramInput):
       self.raiseADebug(" Failed in Node: "+str(xmlNode),verbostiy='silent')
       self.raiseAnError(IOError,'missed subType for the model '+self.name)
 
-  @abc.abstractmethod
+  #@abc.abstractmethod
```

Comment:
why this was commented?

Reply:
I don't recall. I'll uncomment and see what that changes.